src/components/Home/WelcomeHeader.js

```diff
@@ -37,6 +48,94 @@ const WelcomeHeader = () => {
       },
     },
   };
+  
+  useEffect(() => {
+    if (localPath !== ''){
+      const installSampleProtocol = async() => {
+        return openNetcanvas(localPath)
+      }
+      installSampleProtocol()
+    }
+  }, [localPath])
+
+  const getURL = uri =>
+  new Promise((resolve, reject) => {
+    try {
+      resolve(new URL(uri));
+    } catch (error) {
+      reject(error);
+    }
+  });
```

Comment:
I would like you to think about what this code does, and then tell me why you think it is needed.

This is another example of you needing to be more critical about including code from other projects!